public/scripts/components/recipe_form.js

```diff
@@ -34,11 +34,11 @@ var RecipeForm = React.createClass({// eslint-disable-line no-undef
     return {name: '', category: '', ingredients: [{name: '', unit: '', quantity: ''}]} 
   },
   componentWillMount: function () {
-    this.loadUnitsFromServer()
-    this.loadCategoriesFromServer()
+    this.loadUnitsFromServer();
```

Comment:
Good point. At one point I was using a JS syntax checker which kept telling me to remove the ; It was noisey and hard to get used to so I got rid of the checker. I much prefer having semi-colons than not.